### PR TITLE
cache the bvid to vlan translations

### PR DESCRIPTION
**Pull request by aystarik**
Add lookup table for bvid to vlan translations.
bvid_tlb will store previous successful translations from slow  get_vlan_id_from_bvid()
This patch does not change the output from the command, only speeds up it for case of 10k+ MAC tables.

### CODE REVIEW

**Review by prsunny — COMMENTED** [review]
lgtm, request @qiluo-msft to review

File: scripts/fdbshow [hunk 1]
```diff
@@ -111,15 +112,20 @@ class FdbShow(object):
...
-                except Exception:
-                    vlan_id = fdb["bvid"]
-                    print("Failed to get Vlan id for bvid {}\n".format(fdb["bvid"]))
+                bvid = fdb["bvid"]
+                if bvid in bvid_tlb:
+                    vlan_id = bvid_tlb[bvid]
+                else:
+                    try:
+                        vlan_id = port_util.get_vlan_id_from_bvid(self.db, bvid)
+                        if vlan_id is None:
```
Comment (qiluo-msft) [inline scripts/fdbshow:121]: >if vlan_id is None: [](start = 24, length = 19)

Even `if vlan_id is None`, you can still cache it to save future redis access.

**Review by qiluo-msft — COMMENTED** [review]
As comment

File: scripts/fdbshow [hunk 2]
```diff
@@ -87,6 +87,7 @@ class FdbShow(object):
         if not fdb_str:
             return
 
+        bvid_tlb = {}
```
Comment (prsunny) [inline scripts/fdbshow:90]: if you meant bvid table, please rename to `bvid_tbl`

Comment (aystarik) [reply scripts/fdbshow:90]: I meant Translation lookaside buffer, as in MMU, thus TLB.

File: scripts/fdbshow (hunk 1 above)
Comment (aystarik) [reply scripts/fdbshow:122]: Agree, updated patch

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by aystarik** [comment]
@qiluo-msft should I do something to complete those expected checks?

**Review by prsunny — APPROVED** [review]

**Pull request merged**